windows/runner/run_loop.cpp

```diff
@@ -0,0 +1,66 @@
+#include "run_loop.h"
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase